tests/unit/versioned/globber.tap.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+const path = require('path')
+const globber = require('../../../lib/versioned/globber')
+
+const TEST_DIR = path.join(__dirname, 'mock-tests')
+
+tap.test('globber', (t) => {
```

Comment:
I'm not sure but the driver behind all this was because a glob was wrapped in `'` single quotes.  You prob want to add tests for globs that are wrapped in `'` and maybe `"`?

Reply:
Tests added!